const handleError = (err, req, res, next) => {
    console.log(err);
    
}

module.exports = handleError;